amgut/lib/data_access/redcap.py

```diff
@@ -0,0 +1,159 @@
+from urllib import urlencode
+from json import loads
+
+from tornado.web import HTTPError
+from tornado.httpclient import HTTPClient
+
+from amgut.lib.config_manager import AMGUT_CONFIG
+from amgut.lib.data_access.sql_connection import TRN
+
+
+# Explanation of redcap and interactions with it:
+# Redcap has three layers of IDs: record_id, survey_id, and event_id.
+
+# record_id is the id given to a single person in the redcap system, analagous
+# to the host_subject_id.
+
+# survey_id is the name of the survey the person will be taking.
+
+# event_id is the id of the single time a person has taken the survey. It
+# counts up from 1 for each person in each survey. These must be manually
+# created for each survey before the survey is used, so we don't want to waste
```

Comment:
Ah, yes.

@josenavas @antgonza, does this make sense?

@squirrelo, sorry for focusing so much on this. It is obviously generating confusion so just want to make sure we work through it as cleanly as we can so we hate ourselves less in the future